packages/react-query/src/TotalIssuance.tsx

```diff
@@ -5,18 +5,16 @@
 import { BareProps } from '@polkadot/react-api/types';
 
 import React from 'react';
-import { useApi, useCall } from '@polkadot/react-hooks';
 import FormatBalance from './FormatBalance';
+import BN from 'bn.js';
 
 interface Props extends BareProps {
   children?: React.ReactNode;
   label?: React.ReactNode;
 }
 
 export default function TotalIssuance ({ children, className, label, style }: Props): React.ReactElement<Props> {
-  const { api } = useApi();
-  const totalIssuance = useCall<string>(api.query.balances.totalIssuance, []);
```

Comment:
Many places around apps are using `api.query.balances.***` `api.derive.balances.***` in ui, it seems not quite easy to maintain if we do similar changes in other pages as well?

Reply:
Can keep the code commented.. We will need to change all balances stuff as we don't have this module, instead we can replace it with genericAsset at most of the places.. 
Once the design comes, we can think of way to get/store the core asset and then this totalIssuance can take the core asset as params... when we actually start implementing things we can replace the balance calls with ga calls wherever we can..

Reply:
It seems `balances` being able to be added back in future cennznet releases, if in that case, it can be much easier for us
https://github.com/cennznet/cennznet/pull/144/files#diff-a3fea9165475a45ebf09566e2842a9dbR50

Reply:
I believe even if we have balance added back.. we still would like to use GA as we support multi currency module..
@holygits ?

Reply:
correct, we won't use balances module on CENNZnet

Reply:
Yeah... GA is a must I guess. What I mean is that it would be easy to maintain if CENNZnet projects could be generally compatible with Substrate, that is, Cennznet is an extension of Substrate, rather than a total customised version. Otherwise, upgrading gonna be overwhelmed I think. @holygits Any input?

Reply:
An assumption has been made upstream to hard code `balances` in lot's of places so we can't really avoid customizing it.
We'd need to make the balances stuff generic / configurable and contribute that upstream.

In the mean time, I don't think the changes are too hard to handle with good comments and some documentation of the changes